### PR TITLE
Fix tags propogation with seldon client

### PR DESCRIPTION
**Pull request by majolo**
I discovered two small bugs in the seldon client when testing the batch processor. 

* Tags end up being wrapped twice in the response, so the result is `["meta"]["tags"]["tags"]`
* Meta isn't propagated when the gateway is `seldon` not `istio`

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **majolo**
You can assign the PR to them by writing `/assign @majolo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
@majolo: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 4402fb7a1f78a74ea05d26961e74119820c16810 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3374-integration-vltj7) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@majolo: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 4402fb7a1f78a74ea05d26961e74119820c16810 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3374-integration-vltj7) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@majolo: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 4402fb7a1f78a74ea05d26961e74119820c16810 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-3374-integration-65n8b) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by axsaucedo** [comment]
Nice one
/test integration

File: python/seldon_core/batch_processor.py
Comment (RafalSkolasinski) [inline python/seldon_core/batch_processor.py:270]: Is this coming from lint?

**Review by RafalSkolasinski — APPROVED** [review]
Nice one!

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/3374#pullrequestreview-699943828 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: python/seldon_core/batch_processor.py
Comment (majolo) [reply python/seldon_core/batch_processor.py:270]: No it was a warning from my IDE

**Pull request merged**
